psi4/driver/driver_cbs.py

```diff
@@ -647,91 +712,187 @@ def corl_xtpl_helgaker_2(functionname: str, zLO: int, valueLO: float, zHI: int,
+                      valueLO: float,
+                      zHI: int,
+                      valueHI: float,
+                      verbose: bool = True,
+                      alpha: float = None) -> Union[float, core.Matrix, core.Vector]:
+    r"""Alias for the exponential-square root of the extrapolation scheme for reference energies with two adjacent
+    zeta-level bases. Used by :py:func:`~psi4.cbs`.
 
-        return final
+    Parameters
+    ----------
+    functionname
+        Name of the CBS component.
+    zLO
+        Lower zeta level.
+    valueLO
+        Lower value used for extrapolation.
+    zHI
+        Higher zeta level. Should be equal to zLO + 1.
+    valueHI
+        Higher value used for extrapolation.
+    alpha
+        Overrides the default :math:`\alpha = 6.3`
 
-    elif isinstance(valueLO, (core.Matrix, core.Vector)):
+    Returns
+    -------
+    float
+        Returns :math:`E_{total}^{\infty}`, see below.
 
-        beta = valueHI.clone()
-        beta.subtract(valueLO)
-        beta.scale(1 / (zHI**(-alpha) - zLO**(-alpha)))
-        beta.name = 'Helgaker Corl (%s, %s) beta' % (zLO, zHI)
+    Notes
+    -----
+    The extrapolation is calculated according to [3]_:
+    :math:`E_{total}^X = E_{total}^{\infty} + \beta e^{-\alpha\sqrt{X}}, \alpha = 6.3`
 
-        value = valueHI.clone()
-        value.scale(zHI**alpha)
+    References
+    ----------
 
-        tmp = valueLO.clone()
-        tmp.scale(zLO**alpha)
-        value.subtract(tmp)
+    .. [3] Karton, Martin, Theor. Chem. Acc. 115 (2006) 330-333,
+       DOI: 10.1007/s00214-005-0028-6
 
-        value.scale(1 / (zHI**alpha - zLO**alpha))
-        value.name = 'Helgaker Corr (%s, %s) data' % (zLO, zHI)
+    """
 
-        if verbose > 2:
-            core.print_out("""\n   ==> Helgaker 2-point correlated extrapolation for """
-                           """method: %s <==\n\n""" % (functionname.upper()))
-            core.print_out("""   LO-zeta (%s) Data\n""" % (str(zLO)))
-            valueLO.print_out()
-            core.print_out("""   HI-zeta (%s) Data\n""" % (str(zHI)))
-            valueHI.print_out()
-            core.print_out("""   Extrapolated Data:\n""")
-            value.print_out()
-            core.print_out("""   Alpha (exponent) Value:          %16.8f\n""" % alpha)
-            core.print_out("""   Beta Data:\n""")
-            beta.print_out()
+    if type(valueLO) != type(valueHI):
+        raise ValidationError("scf_xtpl_karton_2: Inputs must be of the same datatype! (%s, %s)" %
+                              (type(valueLO), type(valueHI)))
 
+    if alpha is None:
+        alpha = 6.30
 
-#        value.add(valueSCF)
-        return value
+    return xtpl_expsqrt_2(functionname, zLO, valueLO, zHI, valueHI, verbose=verbose, alpha=alpha)
 
-    else:
-        raise ValidationError("corl_xtpl_helgaker_2: datatype is not recognized '%s'." % type(valueLO))
 
-def return_energy_components():
+def scf_xtpl_helgaker_3(functionname: str,
```

Comment:
are these fns duplicated? could you do plain `scf_xtpl_helgaker_3 = xtpl_exponential_3`? or copy `__doc__` if needed?

Reply:
Seconded.